web/themes/custom/novel/templates/views/web/modules/contrib/views_infinite_scroll/templates/views-infinite-scroll-pager.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * The views infinite scroll pager template.
+ */
+#}
+{% if items.next %}
+<ul{{ attributes }}>
+  <li class="result-pager">
```

Comment:
Please consider if it is necessary to wrap the pager in an unordered list.

The contents are not really a list at all. Can it be reduced to a simple `<div>`?